app/editor/src/components/button/styled/Button.tsx

```diff
@@ -22,6 +22,11 @@ export const Button = styled.button<IButtonProps>`
   text-transform: none;
   cursor: pointer;
 
+  & > div {
```

Comment:
default buttons to flex

Reply:
@jdtoombs this may break things.  Not sure of a better way to do this to support loading spinners.

Reply:
Will look into it upon merging!